srtcore/core.cpp

```diff
@@ -8772,8 +8778,8 @@ void srt::CUDT::processCtrlDropReq(const CPacket& ctrlpkt)
             rcvtscc.notify_one();
         }
     }
-
-    dropFromLossLists(dropdata[0], dropdata[1]);
+    // XXX Add some sanity check on dropdata[0]?
+    dropFromLossLists(dropdata[1]);
```

Comment:
Upon the sender's drop request the receiver must drop the specified range of packet sequence numbers or the exact message number. Not all the packets up to the last seqno.

Reply:
Ah, alright. Then I have to prepare this for a possibility to drop either a range, or all up to given sequence, for example by specifying the lower range as SRT_SEQNO_NONE?

Reply:
It should already be working this way.

Reply:
No, so far lower range was always given explicitly and there was no function to remove all sequences less than given one in the receiver loss list. It was in the sender loss list.

Reply:
But those below the specified range must not be removed in the case of the sender's message drop request.
I mean, as of now (current master), the appropriate range is being removed.

Reply:
Exactly, that's why I'm restoring the old version with explicit lower range and for DROPREQ this will remain unchanged. However the +1 -1 problem of including/excluding needs therefore also refax to maintain the old convention of the existing code, so I have to retest it. The function that is exclusively called from TSBPD should still remove all sequences lower than given one, so this will work this way when low = SRT_SEQNO_NONE.